docs/components/movement-sensor/gps/gps-nmea.md

```diff
@@ -11,7 +11,7 @@ aliases:
 ---
 
 A global positioning system (GPS) receives signals from satellites in the earth’s orbit to determine where it is and how fast it is going.
-All supported GPS models provide data for the `Position`, `CompassHeading`, `LinearVelocity`, and `GetAccuracy` methods.
+All supported GPS models provide data for the [`Position`](/components/movement-sensor/#position), [`CompassHeading`](/components/movement-sensor/#compassheading), [`LinearVelocity`](/components/movement-sensor/#linearvelocity), and [`GetAccuracy`](/components/movement-sensor/#getaccuracy) methods.
 You can obtain fix and correction data by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](/components/sensor/).
```

Comment:
```suggestion
You can obtain fix and correction data by using the sensor [`GetReadings`](/components/sensor/#getreadings) method, which is available because GPSes wrap the [sensor component](/components/sensor/).
```